core/src/main/python/update.py

```diff
@@ -44,6 +46,20 @@
 from wlsdeploy.util.model import Model
 from wlsdeploy.util.weblogic_helper import WebLogicHelper
 from wlsdeploy.util import model as model_helper
+# from wlsdeploy.aliases.model_constants import RCU_DB_INFO
```

Comment:
Why is it necessary to leave all these commented sections of code?  Why not remove them?